### ISSUE TITLE
Use pixels unit instead of rem unit to avoid inconsistency due to body font size

### ISSUE THREAD

**Issue by adamsir** [trezor/trezor-ui-components#99]
REM diff
![image](https://user-images.githubusercontent.com/5871877/54156854-5c18b280-4447-11e9-91f3-89259cd8c53a.png)

PX diff (no font-size differences)
![image](https://user-images.githubusercontent.com/5871877/54156890-6f2b8280-4447-11e9-8c06-39b3aebe59fa.png)

**Comment by slowbackspace** [comment]
You need to set the base font-size to 14px (eg. apply it to the body element) if you want the same size as in storybook. Will it solve the problem for you? 

Maybe we should make 16px as a base font-size since that's what all browsers do anyway (or at least mention the 14px font-size in the readme).

We recently migrated from px to rem units in the next wallet. Initially we wanted to use proportional units so the UI can scale with different base font-sizes. They are also used in UI frameworks such as bootstrap and material-ui.
https://engageinteractive.co.uk/blog/em-vs-rem-vs-px
If you feel there are advantages in using px units them we can discuss it further :)

**Comment by adamsir** [comment]
> We recently migrated from px to rem units in the next wallet. Initially we wanted to use proportional units so the UI can scale with different base font-sizes. They are also used in UI frameworks such as bootstrap and material-ui.

Good point. For example Medium use only pixels, which blocks user's browser preferences about font size. Many users do zooming instead of changing the base font size, so REM units are not a big deal about user's accessibility nowadays in my opinion.

My point was in general that If you're writing CSS that's going to be used on a site that you don't control, such as for an embedded widget, there's really no good way to know what apparent size rem will have. 

I'd recommend to use 1rem as base font-size in html element and mention it in the readme if makes sense.
